src/components/button-toggle/button-toggle.style.ts

```diff
@@ -136,7 +138,46 @@ const StyledButtonToggle = styled.button<StyledButtonToggleProps>`
         }
       }
       cursor: not-allowed;
-    `};
+    `}
+
+  ${({ variant, disabled }) =>
+    variant === "minor" &&
+    css`
+      & ${StyledIcon} {
+        color: var(--colorsActionMinor500);
+        :not([aria-pressed="true"]):not(:disabled):hover {
+          color: var(--colorsActionMinorYang100);
+        }
+      }
+
+      color: var(--colorsActionMinor500);
+      border: 1px solid var(--colorsActionMinor500);
+
+      :not([aria-pressed="true"]):not(:disabled):hover {
+        color: var(--colorsActionMinorYang100);
+        background-color: var(--colorsActionMinor600);
+        ${StyledIcon} {
+          color: var(--colorsActionMinorYang100);
+        }
+      }
+
+      ${!disabled &&
```

Comment:
suggestion:  we probably should approach this differently as it's quite confusing

Personally I think any default styles should be applied first i.e. with your `variant === minor` check, and then we override the necessary styles if the `disabled` prop is true. The way we're currently approaching it feels like a bit of a double negative and it makes it more challenging to understand what's going on.